oas/schemas/Statement.yml

```diff
@@ -0,0 +1,16 @@
+title: SCITT Statement
+type: object
+properties:
+  id:
+    type: string
+    example: 'urn:uuid:baaf45fc-5266-11ee-be56-0242ac120002'
+  statement:
+    type: string
+    description: A base64url encoded scitt statement (a cose sign 1, with 🔥 attached payload 🔥)
```

Comment:
This is a problem when your payload is large, which is the case for high resolution images, machine learning models, etc...